boards/nrf52840-dongle/Makefile.dep

```diff
@@ -0,0 +1,25 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+# It'd be nice to pull this in on any stdin dependency and not just shell, but
+# that's the common one, and by the time shell has pulled in stdin, the default
+# stdio_uart has already been set.
+#
+# Maybe have a shell_default like netdev_default?
+ifneq (,$(filter shell,$(USEMODULE)))
+  ifeq (,$(filter stdio_%,$(USEMODULE)))
+    USEMODULE += auto_init_usbus
+    USEMODULE += stdio_cdc_acm
+  endif
+endif
+
+# This will be an additional default netdev to the one set in
```

Comment:
Typically, `gnrc_netdev_default` should only pull in one device. While `usbus_cdc_ecm` is a nice feature, I don't think it is a good idea to enforce it on the user for *every* networking application using (`gnrc_`)`netdev_default`. Except for the "force" argument there are several arguments more that come to mind:

- How should single interface network stacks handle this?
- What do non-routing nodes do with two interfaces, even if the stack is capable to handle them?
- [Hacks like these](https://github.com/RIOT-OS/RIOT/pull/13146/files#diff-da6e86e72e1d67fc548620745ed08db3R31-R35).

All in all it should be the application's decision to include more interfaces (e.g. a border router).

Reply:
I agree, and will roll this back. (There might come boards where USB is the only network interface and thus the default one, like anything that has USB and no wireless, but they're not here yet and they don't have the issues).

I would like to explore how to easily say "but use the host-side network as default instead" (whether host-side is CDC-ECM or ethos) or "and use the host-side network as well", but that doesn't need to be in this PR.

Reply:
> I would like to explore how to easily say "but use the host-side network as default instead" (whether host-side is CDC-ECM or ethos) or "and use the host-side network as well", but that doesn't need to be in this PR.

I'm think that this could be a first step to solve the tooling problems I faced in #13085. However, apart from that, I see this more of a testing tool than a real use case, as nodes (hopefully) don't have to be connected to a host machine in real-world deployment ;-).

Reply:
Yes. Let's keep this out of the board at hand and in the issues it belongs to, now that it's reverted here.